chain/chain/src/resharding/manager.rs

```diff
@@ -80,6 +96,15 @@ impl ReshardingManager {
             vec![split_shard_event.left_child_shard, split_shard_event.right_child_shard],
         )?;
 
+        // Trigger resharding of flat storage.
+        match &self.flat_storage_resharder {
+            Some(resharder) => resharder.start_resharding(
+                ReshardingEventType::SplitShard(split_shard_event.clone()),
+                &next_shard_layout,
+            )?,
+            None => tracing::error!(target: "resharding", "flat storage resharder not initialized"),
```

Comment:
should not we panic if the resharder is not initialized? or it is based on some protocol versioning? if so, it should not error message right?

Reply:
Converting resharding sender to non-option should remove this case.